src/Files.App/Data/Commands/ActionCommand.cs

```diff
@@ -1,8 +1,8 @@
-﻿// Copyright (c) 2024 Files Community
+// Copyright (c) 2024 Files Community
```

Comment:
As above.

Reply:
Probably because I used the online editor.

Reply:
It woun't be problem because VS adds back.

Reply:
As above.